pydarn/plotting/fan.py

```diff
@@ -260,6 +251,9 @@ def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
                 alpha controls the transparency of
                 the fov color
                 Default: 0.5
+            kawrgs: key = value
```

Comment:
Same idea here kwargs could be used for other plotting attributes when using matplot lib